spec/unit/pdk/telemetry_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'spec_helper'
+
+describe PDK::Telemetry do
+  let(:default_config) { {} }
+
+  before(:each) do
+    # We use a hard override to disable analytics for tests, but that obviously
+    # interferes with these tests...
+    ENV.delete('PDK_DISABLE_TELEMETRY')
```

Comment:
Does the ENV get reset somewhere between tests?

Reply:
Yep, in `spec/spec_helper.rb`